### PR TITLE
Review actions info for abstracts and papers

### LINKED ISSUE #3345: List of Abstracts: a column with number of reviews
image

In addition to the column "score" we should have something that indicates the number of reviews and, on hover, some more detailed information such whether the reviews are positive or negative and the suggested contribution types.

### PR DESCRIPTION
**Pull request by mvidalgarcia**
closes #3345

TODO
- [x] Papers

### CODE REVIEW

File: indico/modules/events/abstracts/templates/management/_abstract_list.html [hunk 1]
```diff
@@ -51,6 +52,36 @@
     </div>
 {% endmacro %}
 
+{% macro _render_proposed_contrib_types(review_list) %}
+    {%- for type, reviews in review_list|groupby('proposed_contribution_type') %}
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/management/_abstract_list.html]: {%- for type, reviews in review_list|groupby('proposed_contribution_type') if type %}

and remove the separate `if` below

File: indico/modules/events/abstracts/templates/management/_abstract_list.html [hunk 2]
```diff
@@ -51,6 +52,36 @@
...
+            {{- reviews|selectattr('proposed_contribution_type', 'equalto', type)|list|length }}
+            {{ type.name -}}
+            {{- ", " if not loop.last -}}
+        {% endif -%}
+    {% endfor -%}
+{% endmacro %}
+
+
+{% macro _render_num_reviews_tooltip(abstract) -%}
+        {% if abstract.reviews %}
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/management/_abstract_list.html]: too much indentation

File: indico/modules/events/abstracts/templates/management/_abstract_list.html [hunk 3]
```diff
@@ -118,7 +128,9 @@
                                 </td>
                                 {% for item in static_columns %}
                                     {% if item.id == 'state' %}
-                                        <td class="i-table" data-searchable="{{ abstract.public_state.title | lower }}">
+                                        {% set acceptances = abstract.reviews|selectattr('proposed_action.name', 'equalto', 'accept')|list|length %}
+                                        {% set sort_text = '{}-{}-{}'.format('1' if abstract.is_in_final_state else '0', abstract.state.name, acceptances) %}
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/management/_abstract_list.html]: Does `abstract.is_in_final_state` here actually affect the sorting at all? You already sort by the state afterwards, and `is_in_final_state` only checks for `state != submitted`

Comment (mvidalgarcia) [reply indico/modules/events/abstracts/templates/management/_abstract_list.html]: It can be useful to group abstracts in a final state together.

Comment (ThiefMaster) [reply indico/modules/events/abstracts/templates/management/_abstract_list.html]: Ah, what I originally meant when I suggested grouping them together was to not have "under review" mixed into the list of abstracts in some other state. Anyway, nothing wrong with explicitly having the under-review ones separated from the rest

**Pull request merged**